webservices/args.py

```diff
@@ -231,6 +231,7 @@ def make_seek_args(field=fields.Int, description=None):
 
 totals = {
     'cycle': fields.List(fields.Int, description=docs.RECORD_CYCLE),
+    'type': fields.List(fields.Str, description=docs.COMMITTEE_TYPE)
```

Comment:
I agree with that. But this was to add filtering for whatever model is being hit for that request. For example: at the pac endpoint if ?type='O' is passed in as a query parameter, only committees of that type are returned. It's nothing that was required but more a nice to have, I can take it out if we feel it's not necessary.